src/index.js

```diff
@@ -1,14 +1,40 @@
-import _ from 'lodash';
 import './style.css';
 
-function component() {
-  const element = document.createElement('div');
+const addedItems = document.getElementById('added-items');
+let counter = 0;
 
-  // Lodash, now imported by this script
-  element.innerHTML = _.join(['Hello', 'webpack'], ' ');
-  element.classList.add('hello');
+const todoTasks = [
+  {
+    description: 'wash the dishes',
+    index: 1,
+    completed: false,
+  },
+  {
+    description: 'Complete Todo List Project',
+    index: 2,
+    completed: false,
+  },
```

Comment:
The tasks should appear in order of the index values for each task. Kindly see the screenshot below which shows that objects are being rendered by array index values but actually the requirement is to render them in the order of the index value of an object.
![obj](https://user-images.githubusercontent.com/35504149/171695703-e46d18c3-baff-45b4-89a2-b95995fbca3c.png)
![index](https://user-images.githubusercontent.com/35504149/171695721-f09666f3-20cf-44fe-9c9c-9546e63606ed.png)